.github/workflows/publish_pure_python.yml

```diff
@@ -59,21 +59,9 @@ jobs:
           test_extras: ${{ inputs.test_extras }}
           test_command: ${{ inputs.test_command }}
           pure_python_wheel: true
-      - uses: actions/upload-artifact@v2
-        with:
-          path: dist/*
```

Comment:
Would it be useful to keep the upload-artefact so people can download the wheel even if it isn't uploaded to PyPI? (Also, it should be easy to rebase https://github.com/OpenAstronomy/github-actions-workflows/pull/13 for this PR.)

Reply:
Although happy to merge as I guess this is less important for pure python wheels.

Reply:
People can always build their own wheel but why would they even need it?

Reply:
That's right. Building locally to debug the wheels should be fine. Shall we merge this?